src/CommunityToolkit.Maui.Core/Essentials/SpeechToText/OfflineSpeechToTextImplementation.shared.cs

```diff
@@ -0,0 +1,80 @@
+using System.Globalization;
+using Microsoft.Maui.ApplicationModel;
+
+namespace CommunityToolkit.Maui.Media;
+
+/// <inheritdoc cref="ISpeechToText"/>
+public sealed partial class OfflineSpeechToTextImplementation : ISpeechToText
```

Comment:
This might be by design but I don't think it is currently easy for a developer to register both online and offline speech to text implementations via DI? They would have to register/resolves them as multiples and somehow key them to know which is which. This might be fine. If we have an interface for offline speech to text then we could make that task simpler. Thoughts?

Reply:
They can use keyed service. Updated Sample app